src/game/menu/menu.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import s from "./menu.module.css";
+import close from './close.svg';
+
+export class Menu extends React.Component {
+	constructor(props) {
+		super(props);
+		this.handleClick = this.handleClick.bind(this);
+		this.handleClose = this.handleClose.bind(this);
+	}
+
+	handleClick () {
+		this.props.changeMode();
+	}
+
+	handleClose () {
+		this.props.onClick();
+	}
+
+	render () {
+		let status = this.props.single;
```

Comment:
You can train spread approach here. It will be needed during upcoming project.
For example:
`const{ single } = this.props`

Reply:
Good idea! Thx, but i read, that this approach is called "destructuring assignment", isn't it? 
```suggestion
render () {
	let {single, isMenuOpen} = this.props;
	let buttonName = {single} ? 'SinglePlayer' : 'MultiPlayer';
	let menuClass =  `${styles.menu__container} ${isMenuOpen ? styles.active : ''}`;
```

Reply:
Yes, you are right, its destructuring assignment and its will be used in your project.